routers/users.py

```
@@ -0,0 +1,136 @@
+        # 확장자 체크
+        if ext not in ["jpg", "jpeg", "png"]:
+            validation_errors.append({"field": "profileImage", "issue": "확장자는 jpg, jpeg, png만 가능합니다."})
+        else:
+            # 사이즈 체크
+            try:
+                contents = await profileImage.read()
+                img = Image.open(io.BytesIO(contents))
+                if img.width < 500 or img.height < 500:
+                    validation_errors.append({"field": "profileImage", "issue": "사이즈는 500x500px 이상만 가능합니다."})
+                await profileImage.seek(0)  # 파일 포인터 초기화
+            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)